core/new-gui/src/app/workspace/service/workflow-graph/model/workflow-action.service.ts

```diff
@@ -123,7 +123,10 @@ export class WorkflowActionService {
   private readonly operatorGroup: OperatorGroup;
   private readonly syncTexeraModel: SyncTexeraModel;
   private readonly syncOperatorGroup: SyncOperatorGroup;
-
+  // variable to temporarily hold the current workflow to switch view to a particular version
+  private tempWorkflow = <Workflow>{};
```

Comment:
```suggestion
  private tempWorkflow?: Workflow;
```
Since the workflow could be "empty". it's better to use `undefined` to distinguish an empty workflow and no workflow.

Reply:
thanks made the change